### DISCUSSION TITLE
Queued forever?

### DISCUSSION THREAD

**Discussion by brnsl** [kiwix/apple#446] [Q&A]
Hello, I've noticed that most downloads seem to sit queued forever. I've been trying to download Wikipedia without success. It is not clear why it is queued - it just never starts. I cancelled and was able to immediately download a set of Ted Talks to rule out some other connectivity issue. Is there something else at play here (e.g. the server is not prepared to serve the Wikipedia file (the full ~94GB one))?

Happy to open a proper bug ticket, but this isn't feeling buggy to me since another download succeeded.

Thank you!

**Comment by ghost** [comment]
I have a kinda similar issue. I was able to download any content in Kiwix iOS except Wikipedia itself - each time it starts normally but then stalls on or around 100%.
